docs/conventions/api/README.md

```diff
@@ -1 +1,24 @@
-To Do
+## RDS Query language(RQL) for Filtering, Sorting, and Pagination:
+
+We will be using a query string with a custom format for filtering and sorting, which closely following Github's search formats. You can read more about it [here](https://docs.github.com/en/search-github).
+
+#### Filtering:
+- Single search value: `?q=key:value`
+- Multiple search values: `?q=key:value1+key:value2`
+- Multiple key-value pairs: `?q=key1:value1+key1:value2+key2:value1+key2:value2`
```

Comment:
Can you share an example for the case multiple key value pairs

Reply:
status:assigned status:in-progress

Reply:
But both are different statuses, can they come together?